app/app_configure.go

```diff
@@ -297,7 +317,9 @@ func (app *AkashApp) akashInitGenesisOrder() []string {
 		transfertypes.ModuleName,
 		feegrant.ModuleName,
 		cert.ModuleName,
+		taketypes.ModuleName,
 		escrow.ModuleName,
+		take.ModuleName,
```

Comment:
✂️

Reply:
It complained that I didn't have it in there.  Maybe something else triggered it, will try.

Reply:
ah, you pulled it already.  thanks!